compiler/util/parseJavaScript.js

```diff
@@ -219,19 +218,15 @@ function parseExpression(src, builder, isExpression) {
         jsAST = esprima.parse(src);
     } catch(e) {
         var errorIndex = e.index;
-
-        var errorMessage = e.toString();
+        var errorMessage = '\n' + e.description;
         if (errorIndex != null && errorIndex >= 0) {
-            var srcCharProps = charProps(src);
             if (isExpression) {
                 errorIndex--; // Account for extra paren added to start
             }
-
-            let line = srcCharProps.lineAt(errorIndex)+1;
-            let column = srcCharProps.columnAt(errorIndex)+1;
-            errorMessage += ' [Line ' + line + ', Col: ' + column + ']';
```

Comment:
Hey @yomed, I'll answer since I am reviewing the PR now. The problem is that the line and column number are for the expression only and don't match up with the line and column number within the template. In addition, there is no way to accurately convert the line and column number to a line and column number for the template since the expression gets modified from the original expression in the template to handle expanding placeholders.

Sandeep's solution is to simply show a marker at the column where the error was found on the line with the error. For example:

```
Unexpected identifier: (true sdsds)
                             ^
```